test/test_ops.py

```diff
@@ -24,6 +24,10 @@
 _variant_ops = partial(ops, dtypes=OpDTypes.supported,
                        allowed_dtypes=(torch.float, torch.cfloat))
 
+# Get names of all the operators which have ref in their entry in OpInfo (testing infra)
+# TODO (@krshrimali): Add None to _NOTHING?
+ref_test_ops = list(filter(lambda op: op.ref is not None and op.ref is not _NOTHING, op_db))
```

Comment:
Also filter out the UnaryUfuncInfos, since they're tested against their references in test_unary_ufuncs.py

Reply:
Done, this was done separately but now it's all done in the same place.